operate/qa/migration-tests/migration-test/src/test/java/io/camunda/operate/qa/migration/util/AbstractMigrationTest.java

```diff
@@ -36,7 +55,16 @@
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
-@ContextConfiguration(classes = TestConfig.class)
+@ContextConfiguration(
+    classes = {
```

Comment:
These types of tests where we only load certain classes have been very hard to maintain in the past as new components and dependencies get added. Is there any alternative approach rather than specifying each individual class?

Reply:
Yes, this is not ideal :( The thing is that we don't see much sense in doing unit tests for single classes and methods as the whole logic is in interaction with Elasticsearch and we would like to check behavior of our changes together with Elasticsearch, which brings more classes into play. We also don't want to use the whole Operate application context as then we will have our real Operate indices with much more complex structure and the tests won't be easy to read and maintain. With the current codebase I don't see how we could not list the classes to achieve the desired test behaviour. Do you think we should still switch on starting up the whole application context?